rest/taskana-rest-spring-example-common/src/test/java/pro/taskana/RestHelper.java

```diff
@@ -67,9 +71,13 @@ public static RestTemplate getRestTemplate() {
     mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
     mapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
     mapper.registerModule(new Jackson2HalModule());
+    mapper
+        .registerModule(new ParameterNamesModule())
+        .registerModule(new Jdk8Module())
+        .registerModule(new JavaTimeModule());
```

Comment:
We can inject the ObjectMapper from Spring (so that we don't have to configure the mapper manually)